uint8 num_measurements
float64 position_quality
